lametro/search_indexes.py

```diff
@@ -21,6 +21,7 @@ class LAMetroBillIndex(BillIndex, indexes.Indexable):
     significant_date = indexes.MultiValueField(faceted=True)
     motion_by = indexes.MultiValueField(faceted=True)
     plan_program_policy = indexes.MultiValueField(faceted=True)
+    legislative_session = indexes.MultiValueField(faceted=True)
```

Comment:
This is already defined [on the base index in `django-councilmatic`](https://github.com/datamade/django-councilmatic/blob/5b074f376667766e4b6dbf093871d294bb35fc51/councilmatic_core/haystack_indexes.py#L30) – why redefine here?